tests/utils/test_names.py

```diff
@@ -43,6 +43,9 @@ def test_normalize_name():
     assert normalize_name("Dupont, J.-R.") == "dupont j r"
     assert normalize_name("Dupont") == "dupont"
     assert normalize_name("Dupont J.R.") == "dupont j r"
+    assert normalize_name("von und zu Hohenstein, F.") == "hohenstein f"
```

Comment:
Looks good overall, but could add a couple more tests for affixes? Also we should test when `drop_common_affixes=False`.